### PR TITLE
Update right menu style

### PR DESCRIPTION
**Pull request by gtsiolis**
This will resolve some inconsistencies on the font size used across the top level navigation menus.

Resolve the outstanding points in https://github.com/gitpod-io/gitpod/issues/4471.

| BEFORE | AFTER |
|-|-|
| menu-before |  menu-after |

### CODE REVIEW

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **csweichel** after the PR has been reviewed.
You can assign the PR to them by writing `/assign @csweichel` in a comment when ready.

*No associated issue*. Update pull-request body to add a reference to an issue, or get approval with `/approve no-issue`

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

<details open>
Needs approval from an approver in each of these files:

- **[components/dashboard/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/dashboard/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by gtsiolis** [comment]
@jankeromnes could you take a look at this one-line change since you were involved in https://github.com/gitpod-io/gitpod/issues/4471?

/assign @jankeromnes

**Review by jankeromnes — APPROVED** [review]
Looks good to me! Muchas gracias 🙏

/approve

**Comment by roboquat** [comment]
LGTM label has been added.  <details>Git tree hash: 281bed9460bab0713f92c45ccfeb1fb914eee7d0</details>

**Comment by gtsiolis** [comment]
De nada @jankeromnes! Thanks! ⚾

**Pull request merged**
